proto/viam/app/data/v1/data.proto

```diff
@@ -20,10 +20,17 @@ service DataService {
   rpc BinaryDataByIDs(BinaryDataByIDsRequest) returns (BinaryDataByIDsResponse);
 }
 
+enum Order {
+  ORDER_UNSPECIFIED = 0;
+  ORDER_DESCENDING = 1;
+  ORDER_ASCENDING = 2;
+}
+
 message DataRequest {
   Filter filter = 1;
-  int64 skip = 2;
-  int64 limit = 3;
+  int64 limit = 2;
+  string last = 3;
+  Order sort_order = 4;
```

Comment:
Thoughts on making this an int and reusing the notion of 1 for ascending, -1 for descending, and 0 will be treated as unspecified (natural sort ordering)?

Reply:
I'd prefer an enum just because it limits the number of options they can pass to these 3 that we'd expect, so it would avoid having to do validations anywhere this value is passed